src/snippets/filters/reference/sveltekit/Allow.mdx

```diff
@@ -0,0 +1,13 @@
+import { Code } from "@astrojs/starlight/components";
+import SelectableContent from "@/components/SelectableContent";
+import AllowJS from "./Allow.js?raw";
+import AllowTS from "./Allow.ts?raw";
+
+<SelectableContent client:load frameworkSwitcher syncKey="language">
+  <div slotIdx="1" slot="TS">
+    <Code code={AllowTS} lang="ts" />
```

Comment:
Some of these (like in sveltekit & next) a filename is relevant for routing behavior. Iirc it can be added to the `Code` component